### PR TITLE
Add player samples and favicon in server list ping response

### PR DESCRIPTION
**Pull request by Geolykt**
This PR may be dependent on #20 , however I did not check.

The favicon is currently hardcoded to be server-icon.png, but that will be changeable as soon as we have a proper way for adding configurations :)
Only tested to work with #20 as I'm using Windows 10 1903

### CODE REVIEW

**Comment by ntoskrnl4** [comment]
The default server icon for vanilla/Spigot/Paper is `server-icon.png`, it should be that by default

**Comment by Geolykt** [comment]
Okay, changed

**Review by tazz4843 — APPROVED** [review]

**Review by tazz4843 — CHANGES_REQUESTED** [review]
Please update this PR to use the new code style created in #20 as it changed up the location of a lot of functions, etc.

**Comment by Geolykt** [comment]
This code is severely outdated and I got no time on updating it

File: main.py [hunk 1]
```diff
@@ -17,13 +19,34 @@
...
+logging.info("Started queues!")
+
+# Fundamental MC constants
+totalTime = 0  # Time since the world was created in ticks
+dayTime = 0    # Time of day in ticks
+players = []   # Number of players online
+
+
+def send_task(func, args: list, kwargs: dict, dataOut: multiprocessing.Queue, taskId: int) -> [int, None]:
+    taskData = {'function': func, 'args': args, 'kwargs': kwargs}
```
[the remark this answers is not in the document]
Comment (Geolykt) [reply main.py:43]: I really hate git

**Pull request closed without merging**